src/asRendition.tsx

```diff
@@ -18,6 +18,19 @@ import {
 import { StyledSystemProps } from './common-types';
 import { Tooltips } from './tooltips';
 
+// Returns the display name of a React component. Falls back to 'Component'.
+export const getDisplayName = (component: React.ComponentType): string => {
```

Comment:
I guess that could also be in `reactUtils` right?
I mean, asRendition is already one our most complex files...

Reply:
@thgreasi It's not used anywhere else, so is more likely to rot if it gets moved into a seperate file